docs/change-log/index.md

```diff
@@ -717,38 +717,57 @@ with a drill-down to view the steps and their details. [Tech Preview]
+|ML-4613|UI: The Batch Inference Parameters has incorrect hint on map type. The correct hint is "The 'dict' values should be in JSON key:value format, e.g. {"hello":"world"}" | NA | v1.5.0 |
+|ML-4740|When running function `batch_inference_v2` from the SDK, the `ingest()` function accepts 3 parameters as Data-item or other types: `dataset`, `model_path` and `model_endpoint_sample_set`. If you provided these parameters as non Data-items and later on you want to rerun this function from the UI, you need to provide these parameters as Data-item.|Prepare suitable Data-item and provide it to the batch-rerun UI.| v1.5.0    |
+|ML-4758|In rare cases, deleting a heavy project is unsuccessful and results in a timeout error message while the project moves to offline state.| Delete again.| v1.5.0    |
+|ML-4767|Torch 2.1.0 is not compatible with mlrun-gpu image.              | NA | v1.5.0 |
+|ML-4907|MLRun Client does not support Win OS.                                                 | NA | v1.3.0 | 
+|ML-4942|Dask dashboard does not work on managed clouds.|Dask is externally available thru NodePort, meaning ports needs to be manually open by the infrastructure provider. If running MLRun locally or CE, make sure to port-forward the port Dask Dashboard uses to ensure it is available externally to the Kubernetes cluster. | v1.5.0 |
+|ML-4992|Source target dir ownership is set to root. |Set target dir, and change workdir back: 
+   ```sj.with_source_archive(
+       source=project.source, pull_at_runtime=False, target_dir="/igz/mlrun_code"
+   )
+   sj.spec.build.extra = "WORKDIR /igz"
+   ```
+
 
 ## Limitations
 | ID     |Description                                                                                                                                                                          |Workaround                                                                                                                                                                                                                                                                                                                      |Opened in |
 |---------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|     
+|ML-1278|A user's permission to store data on the underlying storage depends on its V3IO permissions, not on project membership.                                                               |NA                                                               | v0.8.0
```

Comment:
When setting permissions to the project, the right permissions to the data storage need to be set in order to allow access to the data.